Automattic/woocommerce-payments
# fix: Store API tokenized cart PM title

Fixes #

#### Changes proposed in this Pull Request









The payment method title for the tokenized cart feature isn't correct. It results in "Credit card / debit card" instead of "Google Pay"/"Apple Pay".
Fixing.

<img width="335" alt="Screenshot 2024-05-17 at 6 07 52 PM" src="https://github.com/Automattic/woocommerce-payments/assets/273592/5118542b-1e8e-4b18-848d-e4ad4367fc9d">
<img width="1373" alt="Screenshot 2024-05-17 at 6 07 39 PM" src="https://github.com/Automattic/woocommerce-payments/assets/273592/9420b464-8cfe-49e7-b58c-9bfe5029b885">


#### Testing instructions





- Enable the "Enable Cart-Token implementation for PRBs" flag in your dev tools (you'll need to update them)
- Ensure you have Google Pay/Apple Pay enabled in the merchant's settings
- Go to a product page on your site
- Check out with a US address
- The order success page and the order page in the backend should show "Google Pay (WooPayments)" or "Apple Pay (WooPayments)", depending on the selected payment method


-------------------


**Post merge**
